Online-vs-Offline-IDE.md

```diff
@@ -0,0 +1,69 @@
+**\#Online Offline Editors/IDE**
+
+ 
+
+\*\*There are several offline IDE that programmers use in order to stay more
```

Comment:
I am a programmer, and I have no idea what the phrase _Offline IDE_ means. Same with _Online IDE_.

Just so everyone reads this article, is clear what you are talking about, spare a few words on defining such non-standard terms. If that's too much trouble, at least link to some Wikipedia article or any other public text that discusses what these are.